samples/net/syslog_net/README.rst

```diff
@@ -0,0 +1,33 @@
+.. _syslog-net-sample:
+
+Syslog net Application
+######################
+
+Overview
+********
+
+This sample application enables remote syslog service. You need to
```

Comment:
A bit clearer would be:

    This sample application enables a remote syslog service that will
    send syslog messages to a remote server, as configured in ``prj.conf``.